manifests/example-template-windows.yml

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-vm-template
+  annotations:
+    description: "OCP kubevirt windows VM template"
+    tags: "kubevirt,ocp,template,windows"
+  labels:
+    miq.github.io/kubvirt/os: windows-2016
+    miq.github.io/kubevirt-is-vm-template: true
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: ${NAME}
+  spec:
+    template:
+      spec:
+        domain:
+          features:
+            acpi: {}
+            apic: {}
+            hyperv:
+              relaxed: {}
+              vapic: {}
+              spinlocks:
+                spinlocks: 8191
+          clock:
+            utc: {}
+            timer:
+              hpet:
+                present: false
+              pit:
+                tickPolicy: delay
+              rtc:
+                tickPolicy: catchup
+              hyperv: {}
+          cpu:
+            cores: ${CPU_CORES}
+          machine:
+            type: q35
+          resources:
+            requests:
+              memory: ${MEMORY}
+          devices:
+            disks:
+              - name: disk0
+                volumeName: disk0-pvc
+                disk:
+                  dev: sda
+        volumes:
+          - name: disk0-pvc
+            persistentVolumeClaim:
+              claimName: windows-vm-pvc-${NAME}
+- apiVersion: kubevirt.io/v1alpha1
+  kind: PersistentVolumeClaim
+  metadata:
+    name: windows-vm-pvc-${NAME}
+parameters:
+- description: Name for the new VM
+  name: NAME
+- name: MEMORY
+  description: Amount of memory in MB
+  value: 4096Mi
```

Comment:
But the unit is also included in the value.

We need to decide: Either the unit is part of the param (but then the unit can be choosne by the user) or it's part of the template - then it should be set further up.